src/coreclr/inc/corinfo.h

```diff
@@ -2082,9 +2081,16 @@ class ICorStaticInfo
             CORINFO_CLASS_HANDLE        classHnd
             ) { return false; }
 
-    // return the unmanaged calling convention for a PInvoke
-    virtual CorInfoUnmanagedCallConv getUnmanagedCallConv(
-            CORINFO_METHOD_HANDLE       method
+    // return the entry point calling convention for any of the following
+    // - a P/Invoke
+    // - a method marked with UnmanagedCallersOnly
+    // - a function pointer with the CORINFO_CALLCONV_UNMANAGED calling convention.
+    // May return CorInfoCallConvExtension::Managed when the method or call site doesn't
```

Comment:
Change this comment to say that this won't ever return `CorInfoCallConvExtension::Managed` ?